cmake/linux/package_linux.sh.in

```diff
@@ -178,6 +178,16 @@ if [ -e "${APPDIR}/usr/lib/libjack.so.0" ]; then
    ln -sr "${APPDIR}usr/lib/lmms/optional/weakjack.so" "${APPDIR}usr/lib/lmms/optional/libjack.so.0"
 fi
 
+# Use system-provided Carla files
+CARLABIN="/usr/share/carla/resources"
+if [ -d "${CARLABIN}" ]; then
+   rm -f "${APPDIR}/usr/lib/libcarla*.so"
+   mkdir -p "${APPDIR}${CARLABIN}"
```

Comment:
I don't think these are needed (copying carla-plugin startup scripts).
The carla backend will start them using the resource dir, which comes from the system-installed carla.

Reply:
Agreed.  Once I got the bundled libraries removed and `LD_LIBRARY_PATH` set to `/usr/lib/carla`, it worked out.  Removed via 31402e3.